RelationsBll/Services/RelationService.cs

```diff
@@ -24,8 +25,50 @@ public async Task<IEnumerable<Relation>> GetList(Guid? categoryId)
             return await _relations.GetRelationList(categoryId);
         }
 
-        public async Task AddRelation(Relation relation)
+        public async Task AddRelation(AddRelationModel relationModel)
         {
+            Relation relation = new Relation
```

Comment:
use var

Reply:
and can we move it to auto mapper?

Reply:
As i know the Auto-Mapper should stay simple and be spared of any "complex mapping" to different entities that will save important information to data-base; So i decide to do this "logic" by myself, so i can be sure about what is happening and required data goes to proper places;